protocol/scripts/deploys/sepolia/templegold/99-post-create-spice-verify.ts

```diff
@@ -0,0 +1,68 @@
+import '@nomiclabs/hardhat-ethers';
+import '@nomiclabs/hardhat-etherscan';
+import { ethers, run } from 'hardhat';
+import {
+  ensureExpectedEnvvars,
+  mine
+} from '../../helpers';
+import {
+    getDeployedTempleGoldContracts,
+    connectToContracts,
+    ContractInstances
+} from '../../mainnet/templegold/contract-addresses';
+
+async function main() {
+    ensureExpectedEnvvars();
+    const [owner] = await ethers.getSigners();
+    const ownerAddress = await owner.getAddress();
+    const TEMPLEGOLD_ADDRESSES = getDeployedTempleGoldContracts();
+
+    const TEMPLE_GOLD_INSTANCES = connectToContracts(owner);
+    const name = ""; //"TGLD_TOKENNAME_SPICE";
+    const spiceToken = "";
+
+    await mine(TEMPLE_GOLD_INSTANCES.TEMPLE_GOLD.SPICE_AUCTION_FACTORY.createAuction(spiceToken, name));
+    const spiceAuction = await TEMPLE_GOLD_INSTANCES.TEMPLE_GOLD.SPICE_AUCTION_FACTORY.findAuctionForSpiceToken(spiceToken);
+    
+    await run("verify:verify", {
+        address: spiceAuction,
+        constructorArguments: [
+            TEMPLEGOLD_ADDRESSES.TEMPLE_GOLD.TEMPLE_GOLD,
+            spiceToken,
+            ownerAddress,
+            ownerAddress,
+            40161,
+            11155111,
+            "TGLD_TEMPLE_SPICE_2"
+        ],
+    });
+
+    // await _setAuctionConfig(ownerAddress, TEMPLE_GOLD_INSTANCES);
```

Comment:
Are these supposed to be commented out?

Reply:
Didn't want to put a timeout function in there. You can run the two functions immediately if `auctionConfig.startCooldown = 0`